package.json

```diff
@@ -84,6 +84,7 @@
     "eslint-plugin-vue": "^5.2.3",
     "i18next": "^15.0.8",
     "istanbul-lib-coverage": "^2.0.5",
+    "jest-matcher-utils": "^25.1.0",
```

Comment:
Since you removed the references to `jest-matcher-utils`, should this be uninstalled as well?

Reply:
Good catch, thanks